fighters/ganon/src/acmd/other.rs

```diff
@@ -1,279 +1,283 @@
 
 use super::*;
 
-unsafe extern "C" fn damageflyhi_sound(fighter: &mut L2CAgentBase) {
-    let lua_state = fighter.lua_state_agent;
-    let boma = fighter.boma();
+unsafe extern "C" fn sound_damageflyhi(agent: &mut L2CAgentBase) {
+    let lua_state = agent.lua_state_agent;
+    let boma = agent.boma();
     frame(lua_state, 1.0);
-    if is_excute(fighter) {
-        if !StopModule::is_stop(fighter.module_accessor) {
+    if is_excute(agent) {
+        if !StopModule::is_stop(agent.module_accessor) {
             let play_vc = if DamageModule::reaction(boma, 0) < 100.0 {
-                app::sv_math::rand(hash40("fighter"), 3)
+                app::sv_math::rand(hash40("agent"), 3)
```

Comment:
this hash used in the `rand` call should remain as "fighter". same apples to other instances in this file

(i see you did this for dedede as well, so i'll fix that for you since it's already merged)